tests/integration_tests/test_multi_2_sec_AS.py

```diff
@@ -0,0 +1,379 @@
+from openmdao.utils.assert_utils import assert_near_equal
+import unittest
+
+
+class Test(unittest.TestCase):
```

Comment:
I think it helps to add comments or docstrings briefly explaining what you test in this test file (like you compare optimized results to a ref value and also check that construction and constraint approach give the same result)